IntrepidSwiftWisdomTesting.podspec

```diff
@@ -1,17 +1,17 @@
 Pod::Spec.new do |s|
   s.name          = "IntrepidSwiftWisdomTesting"
-  s.version       = "0.0.7"
+  s.version       = "0.7.3"
```

Comment:
It was a conscious decision to keep this in lockstep with the Intrepid.podspec version. Please let me know if it's better for this to have its own separate version.